docs/how-to/resolve-github-connection-errors.md

```diff
@@ -0,0 +1,47 @@
+---
+id: resolve-github-connection-errors
+title: How to Resolve GitHub Connection Errors in Amplication
+sidebar_label: How To Resolve GitHub Connection Errors
+slug: /how-to/resolve-github-connection-errors
```

Comment:
Not so sure about it, but-
Do you think this page should be in the "Sync With Git" directory (as part of the docs), or as part of the "Troubleshooting" --> "GitHub" directory?